### PR TITLE
CNI Beta Graduation.

### PR DESCRIPTION
**Pull request by bianpengyuan**
fixes https://github.com/istio/enhancements/issues/86

Rationale for the beta promotion:
* TOC approved promotion plan is completed: https://docs.google.com/document/d/1RMGiH-xFmmnct7bkBgmESQ4FONNuZ1J1V4oWQ8aB5qk/edit# which includes document revamp & tests.
* Supportability review has been performed, and conclusion is we should proceed with several improvements to the new CNI doc: https://docs.google.com/document/d/1V8L1nK_rPSAlzUCfeVFO9NELz31lsUPyW4o9O1j3Q8I/edit?ts=60fed4dc#heading=h.at93znnl88pm
* CNI related github issues has been reviewed and there is no critical bug or feature gap left.

### CODE REVIEW

**Review by howardjohn — COMMENTED** [review]
LGTM, will let others take a look

**Comment by bianpengyuan** [comment]
@istio/technical-oversight-committee please take a look and let me know if there is anything missed. Thanks!

**Comment by brian-avery** [comment]
https://github.com/istio/enhancements/blob/master/features.yaml will need to be updated before this can be promoted and that will trigger TOC as codeowners

File: features.yaml [hunk 1]
```diff
@@ -269,7 +263,7 @@ features:
     link: "/docs/setup/additional-setup/cni/"
     level:
       checklist: ""
```
Comment (brian-avery) [inline features.yaml:265]:
```suggestion
      checklist: "features/cni.md"
```

**Comment by bianpengyuan** [comment]
@istio/technical-oversight-committee please take a look, thanks a lot!

**Comment by bianpengyuan** [comment]
@nrjpoddar @smawson @linsun @louiscryan @brian-avery Any comments/opinions on the promotion? The TOC approved plan has been fully executed and we are close to 1.11 release date. Really hope to get this in if possible..

**Review by howardjohn — APPROVED** [review]

**Comment by nrjpoddar** [comment]
LGTM, thanks for all the work on this!

**Comment by ericvn** [comment]
Note that this needs to be cherrypicked to 1.11 so it's picked up in the docs (currently preliminary.istio.io and soon istio.io (after the branch cut/release)).

**Pull request merged**